### PR TITLE
Update method for setting filter descriptions.

### PR DESCRIPTION
**Pull request by dekoza**
I want to merge this change because there was no way to set description for complex filters - the ones that require descriptions the most.

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [x] Privileged queries and mutations are guarded by proper permission checks
* [x] Database queries are optimized and the number of queries is constant
* [x] Database migration files are up to date
* [x] The changes are tested
* [x] GraphQL schema and type definitions are up to date
* [x] Changes are mentioned in the changelog

### CODE REVIEW

File: saleor/graphql/product/filters.py [hunk 1]
```diff
@@ -585,6 +585,7 @@ class ProductFilter(MetadataFilterBase):
         input_class=PriceRangeInput,
         method="filter_minimal_price",
         field_name="minimal_price_amount",
+        help_text="Product's minimal price (after discounts) is in given range.",
```
Comment (fowczarek) [inline saleor/graphql/product/filters.py:588]: Maybe
"Filter by lowest variant price after discounts"

File: saleor/graphql/core/types/converter.py [hunk 2]
```diff
@@ -17,7 +17,10 @@
 
 
 def get_form_field_description(field):
-    return str(field.help_text) if field.help_text else None
+    if hasattr(field, "help_text"):
```
Comment (fowczarek) [inline saleor/graphql/core/types/converter.py:20]: IMO we could use `description` as the name of this field.

Comment (dekoza) [reply saleor/graphql/core/types/converter.py:20]: `help_text` is standard attribute for Django models and thus it's somewhat familiar. I also saw it used frequently in saleor's filter definitions. I think changing that notation is out of scope here.

**Review by IKarbowiak — APPROVED** [review]

File: saleor/graphql/product/filters.py (hunk 1 above)
Comment (dekoza) [reply saleor/graphql/product/filters.py]: Done

**Review by maarcingebala — APPROVED** [review]

**Pull request merged**